app/retail/emails.py

```diff
@@ -1213,9 +1256,14 @@ def resend_new_tip(request):
 @staff_member_required
 def new_bounty(request):
     from dashboard.models import Bounty
+    from marketing.views import quest_of_the_day, upcoming_grant, upcoming_hackathon, latest_activities
     bounties = Bounty.objects.current().order_by('-web3_created')[0:3]
     old_bounties = Bounty.objects.current().order_by('-web3_created')[0:3]
-    response_html, _ = render_new_bounty(settings.CONTACT_EMAIL, bounties, old_bounties, int(request.GET.get('offset', 2)))
+    quest = quest_of_the_day()
```

Comment:
you don't need to create temp variables in each step here if its just calling the function, just use the function calls as arguments to render_new_bounty

Reply:
Done in [fdc927f](https://github.com/gitcoinco/web/pull/6471/commits/fdc927f9b608a92c69079aea7041c571d03c344d)